src/main/scala/com/campudus/tableaux/router/RouterRegistry.scala

```diff
@@ -28,9 +28,8 @@ object RouterRegistry extends LazyLogging {
 
     val mainRouter: Router = Router.router(vertx)
 
-    implicit val requestContext: RequestContext = RequestContext()
     mainRouter.route().handler(CookieHandler.create())
-    mainRouter.route().handler(retrieveCookies)
+    // mainRouter.route().handler(retrieveCookies)
```

Comment:
**[major]** Should this be commented code? If we don't need this anymore the code should be removed?